src/components/WizardModal/WizardModal.docs.mdx

```diff
@@ -0,0 +1,12 @@
+import { Meta, Story } from '@storybook/addon-docs/blocks';
+import { Figma } from 'storybook-addon-designs/blocks'
+
+<Meta title="components/modals/SemanticModal/Docs" />
+
+# Modal documentation
+
+>**[Confluence page](https://zitenote.atlassian.net/l/c/ur1JbSHg)** about pattern usage, rationale, planned changes, and any current discrepancies between Storybook and Figma.
+
+> **Visual documentation**: please open in Figma for full context, these are only the most prominent examples of the pattern.
+
+<Figma url="https://www.figma.com/file/i13gqrwTYjmMhHkEcwaM3x/Persona-based-Onboarding?node-id=279%3A8312"/>
```

Comment:
I would replace this link with https://www.figma.com/file/vnzYfrwIISRYkr2iYE5TcD/DS-Atoms?node-id=24200%3A24909